### ISSUE TITLE
Detect Song/Artist language

### ISSUE THREAD

**Issue by Kiina** [simon-weber/Autoplaylists-for-Google-Music#190]
Okay this might be a bit of a stretch but I've always been annoyed by how I can't easily only play songs/albums in a specific language on Google play music. I though of many ways and probably the only way I could reasonably approach this issue is by scraping wikipedia or other metadata providers for example, make a big list with all artists and feed it into the: artist contains. 
But the current layout makes it a bit impossible. Not sure if this would be wise from a performance standpoint but having a field that goes through line for line and check if  its one of the artists, separated by a newline.

So basically provide a readable array to artist contains or a better way to get the language of a song (which isn't provided anywhere in the metadata except maybe an educated guess from all the song titles).

**Comment by simon-weber** [comment]
Hm, interesting. You could maybe hack the big-list approach by generating a massive regex - eg `artist matches regex "artist1|artist2|..."`, though your computer might blast off trying to run the query, haha.

I can double check if there's any mention of language in Google's hidden metadata, but I think you're right that they don't provide it.

I think the ideal approach would be to make an external thing (probably a gmusicapi script) that adds the language into the comment field, then have Autoplaylists query on eg `comment contains "lang:english"`.
